### DISCUSSION TITLE
在edge浏览器下极简模式下呼出界面困难

### DISCUSSION THREAD

**Discussion by vurelee** [xiaoshuapp/explorer-xiaoshu#564] [Feedback]
edge浏览器网页边上有空白。所以极简模式下呼出工具栏有点困难。
![呼出](https://github.com/xiaoshuapp/explorer-xiaoshu/assets/18027005/814ebb9e-5fef-459e-b2f9-b2ac3505c5fc)

**Comment by nexmoe** [comment]
近期会考虑优化
